### PR TITLE
docs: remove `unstable`

### PR DESCRIPTION
**Pull request by NekoChanTaiwan**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

remove `unstable` in other languages.

### CODE REVIEW

**Comment by c-ehrlich** [comment]
Wow the notifications are working :)

**Comment by AmadeusTwi** [comment]
While this is a good edit, I also noticed that the bulletpoint was expanded with some new text in English docs.
I'm not sure if we should delete "unstable" in this PR in multiple languages or encourage just plain updating the page itself in these languages.

File: www/src/pages/pt/usage/next-auth.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Ao usar NextAuth.js com tRPC, você pode criar procedimentos protegidos e reutil
 
 Isso é feito em um processo de duas etapas:
 
-1. Pegar a sessão dos cabeçalhos de solicitação usando a função [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). Não se preocupe, esta função é segura de usar - o nome inclui `unstable` apenas porque a implementação da API pode mudar no futuro. A vantagem de usar `getServerSession` em vez do `getSession` regular é que é uma função somente do lado do servidor e não aciona chamadas de busca desnecessárias. `create-t3-app` cria uma função auxiliar que abstrai essa API peculiar.
+1. Pegar a sessão dos cabeçalhos de solicitação usando a função [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). A vantagem de usar `getServerSession` em vez do `getSession` regular é que é uma função somente do lado do servidor e não aciona chamadas de busca desnecessárias. `create-t3-app` cria uma função auxiliar que abstrai essa API peculiar.
```
Comment (Sn0wye) [inline www/src/pages/pt/usage/next-auth.md:74]:
```suggestion
1. Pegar a sessão dos headers da request usando a função [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). A vantagem de usar `getServerSession` em vez do `getSession` regular é que é uma função que é executada somente do lado do servidor e não faz chamadas de busca desnecessárias. O `create-t3-app` cria uma função auxiliar que abstrai essa API peculiar.
```

**Review by Sn0wye — CHANGES_REQUESTED** [review]
That's it.

**Comment by Sn0wye** [comment]
> While this is a good edit, I also noticed that the bulletpoint was expanded with some new text in English docs. I'm not sure if we should delete "unstable" in this PR in multiple languages or encourage just plain updating the page itself in these languages in multiple PRs, since it requires a lot of translators at once. Opinions?

Looks great, if the people involved come to an agreement, I'm in.

File: www/src/pages/pl/usage/next-auth.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Jeżeli używasz NextAuth.js oraz tRPC, stworzyć można zabezpieczone procedury
 
 Konfiguracja ta zachodzi w dwóch krokach:
 
-1. Pobierz sesję z headerów zapytania korzystając z funkcji [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). Nie martw się, funkcja ta jest bezpieczna do użycia - nazwa zawiera słowo `unstable` jedynie dlatego, ponieważ implementacja API w przyszłości może się zmienić. Zaletą korzystania z `getServerSession` zamiast `getSession` jest fakt, iż jest to funkcja wywoływana jedynie po stronie serwera i nie inicjuje ona żadnych niepotrzebnych zapytań. `create-t3-app` tworzy funkcję pomocniczą, która ułatwia korzystanie z `getServerSession`.
```
Comment (PiotrekPKP) [inline www/src/pages/pl/usage/next-auth.md:94]: Looks good :+1:

Comment (matibox) [reply www/src/pages/pl/usage/next-auth.md:94]: Yup, can confirm that

File: www/src/pages/ar/usage/next-auth.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ declare module "next-auth" {
 
 يُمكن فَعل هذا في خطوتين:
 
-1. للحصول علي Object الـ Session يمكنك استخدام getServerSession، لا تقلق فهي امنه unstable تعني انها يمكن ان تتغير في المستقبل.
+1. للحصول علي Object الـ Session يمكنك استخدام getServerSession،
```
Comment (Sboonny) [inline www/src/pages/ar/usage/next-auth.md:73]: LGTM 🎉

image

**Review by AmadeusTwi — APPROVED** [review]
Russian LGTM.

File: www/src/pages/fr/usage/next-auth.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ Lorsque vous utilisez NextAuth.js avec tRPC, vous pouvez créer des procédures
 
 Cela se fait en deux étapes :
 
-1. Récupérez la session à partir des en-têtes de requête à l'aide de la fonction [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). Ne vous inquiétez pas, cette fonction est sûre à utiliser - le nom inclut "unstable" uniquement parce que l'implémentation de l'API peut changer à l'avenir. L'avantage d'utiliser `getServerSession` au lieu de `getSession` est qu'il s'agit d'une fonction côté serveur uniquement et qu'elle ne déclenche pas d'appels de récupération inutiles. `create-t3-app` crée une fonction d'assistance qui résume cette API particulière.
+1. Récupérez la session à partir des en-têtes de requête à l'aide de la fonction [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession). L'avantage d'utiliser `getServerSession` au lieu de `getSession` est qu'il s'agit d'une fonction côté serveur uniquement et qu'elle ne déclenche pas d'appels de récupération inutiles. `create-t3-app` crée une fonction d'assistance qui résume cette API particulière.
```
Comment (joachimjusth) [inline www/src/pages/fr/usage/next-auth.md:97]: Ok for me :)

**Review by joachimjusth — APPROVED** [review]

**Review by lodisy — APPROVED** [review]
👌👌

**Review by c-ehrlich — APPROVED** [review]
Looks good, thanks for updating this :)

> While this is a good edit, I also noticed that the bulletpoint was expanded with some new text in English docs.
I'm not sure if we should delete "unstable" in this PR in multiple languages or encourage just plain updating the page itself in these languages in multiple PRs, since it requires a lot of translators at once.
Opinions?

In this case since it has already been approved for all languages but one let's get it merged. But I agree that in general PRs should not touch multiple languages of docs unless it's just a change to a code block. In the future I'll probably close these and ask for separate PRs to be opened for each language.

**Pull request merged**